official/vision/beta/projects/panoptic_maskrcnn/configs/panoptic_maskrcnn.py

```diff
@@ -72,6 +73,18 @@ class DataConfig(maskrcnn.DataConfig):
   parser: Parser = Parser()
 
 
+@dataclasses.dataclass
+class PanopticSegmentationGenerator(hyperparams.Config):
+  output_size: List[int] = dataclasses.field(
+      default_factory=list)
+  stuff_classes_offset: int = 0
```

Comment:
I feel this offset is important even in the absence of the panoptic generator, right? can we have it as part of task/model.